README.md

```diff
@@ -19,6 +19,17 @@ The new and improved DDW Analyst UI interface
 11. Bundle frontend code and collect static files `npm run build`
 12. Restart the app. `docker-compose restart`
 
+### Import CSV Files
+
+1. Manually run management command to download csv files:
+
+        docker-compose exec web python manage.py update_csv_files
+
+2. Create a scheduled event to periodically download updates from the git repo. The bash script is `update_csv_files.sh`
+
+3. The git hub repo with csv files can be found here https://github.com/devinit/ddw-data-update-configs
```

Comment:
Put the link in the `here` string ... i.e [here](https://github.com/devinit/ddw-data-update-configs)

Reply:
Updated